examples/common/scripts/vttablet-up.sh

```diff
@@ -56,6 +56,7 @@ vttablet \
  --pid_file $VTDATAROOT/$tablet_dir/vttablet.pid \
  --vtctld_addr http://$hostname:$vtctld_web_port/ \
  --disable_active_reparents \
+ --throttler-config-via-topo --heartbeat_enable --heartbeat_interval=250ms --heartbeat_on_demand_duration=5s \
```

Comment:
nit: we can remove `--enable_replication_reporter`. `--heartbeat_enable` supersedes it.

Reply:
OK! I'll do that before merging.